spec/mailers/user_mailer_spec.rb

```diff
@@ -40,17 +40,23 @@
 
   describe 'reservation_status_update' do
     it 'sends to the reserver' do
+      # make it a request so the email sends
+      @res.update_attributes(flags: Reservation::FLAGS[:request])
```

Comment:
I'm not happy with this, but I also don't want to make the default reservation created before each test something other than a basic valid reservation